File: cfgs/java_cfgs_foo/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int count = 0;15;
4;statement;int max_count = 0;15;
5;statement;int min_count = foo_1;15;
6;statement;int i = 0;16;
7;conditional;foo_1 < (foo_2 - 1);17;
8;conditional;foo_1[foo_2] == foo_3[foo_4 + 1];19;
9;statement;count += 1;20;
10;statement;continue;21;
11;statement;max_count = fun_1(foo_1, foo_2);24;
12;statement;min_count = fun_1(foo_1, foo_2);25;
13;statement;count = 0;26;
14;statement;i++;18;
15;statement;return foo_1 - foo_2;29;
16;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;
6;8;15;
7;9;11;
8;10;
9;7;
10;12;
11;13;
12;14;
13;7;
14;16;
15;

File: cfgs/pythoncfgs/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY.py
Nodes
1;statement;fun_1();8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = 0;9;
4;statement;foo_1 = foo_2;9;
5;conditional;foo < foo;10;
6;statement;return foo_1 - foo_2;18;
7;conditional;foo_1[foo_2] == foo_3[foo_4 + 1];11;
8;statement;foo_1 = fun_1(foo_2, foo_3);15;
9;statement;foo_1 = fun_1(foo_2, foo_3);16;
10;statement;foo_1 = 0;17;
11;statement;foo_1 += 1;12;
12;exit;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;7;
5;12;
6;8;11;
7;9;
8;10;
9;5;
10;5;
11;